packages/wp-now/src/download.ts

```diff
@@ -95,7 +95,9 @@ export async function downloadWordPress(
 	});
 	if (downloaded) {
 		fs.ensureDirSync(path.dirname(finalFolder));
-		fs.renameSync(path.join(tempFolder, 'wordpress'), finalFolder);
+		fs.moveSync(path.join(tempFolder, 'wordpress'), finalFolder, {
+			overwrite: true,
+		});
```

Comment:
This solves @kozer  issue https://github.com/WordPress/wordpress-playground/pull/336#issuecomment-1547900219